src/status_im/common/controlled_input/utils.cljs

```diff
@@ -78,16 +78,19 @@
 
 (defn set-upper-limit
   [state limit]
-  (when limit
+  (if limit
```

Comment:
Without this change `set-upper-limit` breaks the usage of `->` because no value returned when limit not applied